### ISSUE TITLE
Release jinja2-slug 0.3.0

### ISSUE THREAD

**Issue by barraponto** [xsteadfastx/jinja2-slug#9]
Because jinja2-slug 0.2.0 does not support Jinja 3, we need another version released to PyPI.
We could also take the time to fill in more details for the PyPI page, it looks totally empty now: https://pypi.org/project/jinja2-slug/

**Comment by xsteadfastx** [comment]
right now im pretty unsusre what to do with the project. im kind of out
of the python world for a few years now. i was never really sure this is
usable to some people. so i wonder how we can keep this alive if its
really used by people.

**Comment by barraponto** [comment]
I arrived here from [Copier docs](https://copier.readthedocs.io/en/stable/configuring/#jinja_extensions). It is only suggested there, but I really wanted to make jinja2-slug a default extension in copier, since it makes a good job of integrating unicode-slugify to jinja.

I'd love to provide more support for the plugin as well.

**Comment by wwuck** [comment]
@xsteadfastx Would you consider donating the project to the Pallets community ecosystem?  Seems appropriate since jinja is a Pallets project.

https://palletsprojects.com/ecosystem

https://github.com/pallets-eco/

**Comment by xsteadfastx** [comment]
yes im willing to donate. but how does this work? are actually people using this?

**Comment by wwuck** [comment]
Yes, I would be using this if it were updated and released to pypi. The current release on pypi has a restriction of `jinja2 (>=2.10,<3.0)` so it's not compatible with the latest `copier` which requires `jinja2 = ">=3.1.5"`.

I'm not associated with pallets project, but from a quick read of their docs it looks like you would ask to join the community on [their discord server](https://discord.gg/pallets) and then transfer the project inside github after receiving the github org invite. I'm guessing that may also involve transfer of pypi project ownership somehow? Not really sure on that part.
